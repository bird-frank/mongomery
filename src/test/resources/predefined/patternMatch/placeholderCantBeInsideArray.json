{
  "TestCollection": [
    {
      "_id": {
        "$oid": "55f92b343944ec3f14a94459"
      },
      "emails": [
        "firs@email.com",
        "second@email.com"
      ],
      "firstName": "firstName"
    }
  ]
}


