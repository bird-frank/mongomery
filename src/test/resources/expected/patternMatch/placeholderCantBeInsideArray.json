{
  "TestCollection": [
    {
      "_id": {
        "$oid": "55f92b343944ec3f14a94459"
      },
      "emails": [
        "$anyString()",
        "second@email.com"
      ],
      "firstName": "firstName"
    }
  ]
}


